src/compiler/transformers/transform-utils.ts

```diff
@@ -705,19 +711,19 @@ export const createImportStatement = (importFnNames: string[], importPath: strin
     );
   });
 
-  return ts.createImportDeclaration(
+  return ts.factory.createImportDeclaration(
     undefined,
     undefined,
-    ts.createImportClause(undefined, ts.createNamedImports(importSpecifiers)),
-    ts.createLiteral(importPath)
+    ts.factory.createImportClause(false, undefined, ts.factory.createNamedImports(importSpecifiers)),
```

Comment:
We add `false` here because the function signature has changed such that `isTypeOnly` is no longer optional, is typed as `boolean`, and is now the first parameter to the function:
```diff
- createImportClause(isTypeOnly: boolean, name: Identifier | undefined, namedBindings: NamedImportBindings | undefined): ImportClause;
+ createImportClause(name: Identifier | undefined, namedBindings: NamedImportBindings | undefined, isTypeOnly?: any): ImportClause;
```